documentation/release_notes.rst

```diff
@@ -8,6 +8,59 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.6
+=============
+
+New Features
+------------
+- Added one more thread parallel backend. It is based on OpenMP 5.0. To enable the OpenMP backend, please define macro ``ONEDPL_USE_OPENMP_BACKEND`` into 1.
+- Added the range-based version of the ``reduce_by_segment`` algorithm and improved performance of the iterator-based ``reduce_by_segment`` APIs.
+  Please note that the use of the ``reduce_by_segment`` algorithms requires C++17.
+- Added the serial-based versions of the following algorithms: ``all_of``, ``any_of``, 
+  ``none_of``, ``count``, ``count_if``, ``for_each``, ``find``, ``find_if``, ``find_if_not``.
+  For the detailed list, please refer to `Tested Standard C++ API Reference`_. 
+
+Fixed Issues
+------------
+- Fixed hangs and errors when oneDPL is used together with oneMKL in DPC++ programs.
+- Fixed possible data races in the following algorithms used with DPC++ execution policies: ``sort``, ``stable_sort``, ``partial_sort``,``nth_element``.
+- Fixed ``param_type`` API of random number distributions to satisfy C++ ``RandomNumberDistribution`` named requirement.
+  Please note that the changes may cause API/ABI backward compatibility issues in cases when ``param_type`` is explicitly used.
+
+Known Issues and Limitations
+----------------------------
+- No new issues in this release. 
+
+Existing Issues
+^^^^^^^^^^^^^^^
+- The definition of lambda functions used with parallel algorithms should not depend on preprocessor macros
+  that makes it different for the host and the device. Otherwise, the behavior is undefined.
+- ``exclusive_scan`` and ``transform_exclusive_scan`` algorithms may provide wrong results with vector execution policies
+  when building a program with GCC 10 and using -O0 option.
+- Some algorithms may hang when a program is built with -O0 option, executed on GPU devices and large number of elements is to be processed.
```

Comment:
I was not able to reproduce the hang with the latest Intel(R) Graphics Compute Runtime and Intel® oneAPI DPC++/C++ Compiler. I suggest removing the note as not relevant one.

```suggestion
```